openmdao/test_suite/components/paraboloid.py

```diff
@@ -16,6 +16,9 @@ def setup(self):
 
         self.add_output('f_xy', val=0.0)
 
+        # Finite difference all partials.
+        self.approx_partials('*', '*', method='fd')
+
     def compute(self, inputs, outputs):
         """
```

Comment:
Need to make your own test component rather than modifying this, since this one is used a bunch of places for specific purposes. Then you can get rid of the changes in paraboloid_mat_vec.